src/dune_digest/dune_digest.ml

```diff
@@ -66,13 +66,17 @@ let generic a =
   Metrics.Timer.record "generic_digest" ~f:(fun () ->
       string (Marshal.to_string a [ No_sharing ]))
 
-let file_with_executable_bit ~executable path =
+let path_with_executable_bit =
```

Comment:
What is the point of this renaming? It seems like if we speak of executable bits, we're speaking of files.

Reply:
Directories also have executable bits though. And on macos and freebsd, so can symlinks.

Reply:
Right. Thanks, makes sense.